src/components/contact-info/contact-info.js

```diff
@@ -9,8 +9,8 @@ const ContactInfo = ({ onClick, profileName, alumni }) => {
         for you?
       </h3>
       <p>
-        Contact {profileName ? `${profileName}'s` : 'an'} Account Manager to
-        discuss your project
+        Contact {profileName && !alumni ? `${profileName}'s` : 'an'} Account
+        Manager to discuss your project
```

Comment:
Hey @tiagov8, 

Which do you like better? 

Option 1. 
> Contact **SSW's** Account Manager to discuss your project

Option 2. 
> Contact **an** Account Manager to discuss your project

Reply:
Option 3.

> Contact an SSW Account Manager to discuss your project

Reply:
CC: @tiagov8 @JackDevAU 

As per my conversation with @JackDevAU, we will go with option 3.

Resolved in https://github.com/SSWConsulting/SSW.People/pull/608/commits/cfb8d48e2829f7e20a7a766810c22bf35a6841e8